src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -53,7 +53,7 @@ public Integer getEntryNumber() {
 
     @JsonProperty("item-hash")
     public String getItemHash() {
-        return "sha-256:" + sha256hex;
+        return hashValue.toString();
```

Comment:
That's fine when we're referencing `item-hash`, but if we ever call `getSha256hex()` from code we're returning a `HashValue` object, as opposed to a `String`, so we'll need to explicitly call `toString()` if we want to compare this `HashValue` to a `String`.